README.md

```diff
@@ -253,8 +253,9 @@ component's "real" props. In this example lets use `_testID`.
 ```js
 let Greeting = props => <div>hello <strong _testID='name'>{props.name}</strong></div>;
 
-$(Greeting).render()
-  .prop({ name: 'Betty' })
+$(<Greeting />)
```

Comment:
Great, sorted my side too.

Thanks for this awesome library.  I've been waiting a long time for this to come along.